go-controller/pkg/crd/egressqos/v1/types.go

```diff
@@ -71,8 +72,14 @@ type EgressQoSRule struct {
 
 // EgressQoSStatus defines the observed state of EgressQoS
 type EgressQoSStatus struct {
-	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	// A concise indication of whether the EgressQoS resource is applied with success.
+	// +optional
+	Status string `json:"status,omitempty"`
+	// An array of Human-readable messages indicating details about the status of the object.
```

Comment:
can I get an example of how this will look like?
is it:
Status: Error
Messages: {"libovsdb error", "add error", "blah} ?

why are there no limits set to this array? Shouldn't we be using some kinda of MaxProperties?

Reply:
I think we must not set limit for array, it would be equivalent to ic zone size.

This is how it looks like for IC and Non IC zone:

```
# kubectl get egressqos
NAME      STATUS
default   EgressQoS Rules applied
```

```
  status:
    messages:
    - 'ovn-worker: EgressQoS Rules applied'
    - 'ovn-control-plane: EgressQoS Rules applied'
    - 'ovn-worker2: EgressQoS Rules applied'
    - 'ovn-worker3: EgressQoS Rules applied'
    status: EgressQoS Rules applied
```
    
```
  status:
    messages:
    - 'global: EgressQoS Rules applied'
    status: EgressQoS Rules applied
```